specs/pool_list_view.adoc

```diff
@@ -0,0 +1,147 @@
+// vim: tw=79
+
+= Tendrl Definitions: Display list of pool present in Ceph cluster in Tendrl
+
+Display list of pool associated with a ceph cluster present in Tendrl. The list should have functionality for 
+
+* filtering by pool name, cluster name, status and type.
+* sorting by pool name.
+
+Pool list will be sorted by name by default.
+
+== Problem description
+
+Tendrl should show list of pool associated with a ceph cluster along with their details.
+
+The details shown for each pool in UI is -
+
+* Pool Name
+* Status
+* Cluster Name
+* Pool Type
+* CPU Utilisation
+* Replicas
+* OSDs
+* PGs
+* Quotas
+* Active alerts
+
+API support present for -
+
+* Pool Name
+* Status
+* Cluster Name
+* CPU Utilisation
+* Replicas
+* OSDs
+* PGs
+
+== Use Cases
+
+Users can see the list of pool associated with a ceph cluster present in Tendrl. They can also 
+
+* filter by pool name, cluster name, status and type.
+* sort by pool name.
+
+As per API support, users can 
+
+* filter by pool name, cluster name and status.
+* sort by pool name.
+
+== Proposed change
+
+* Create a view for listing pool associated with a ceph cluster.
+* For utilisation, use donut chart component.
+* Implement client side filer and sorting functionality.
+* Filter call will trigger after 1 sec the user enters the filter data.
+* Send polling call after 10 minutes to get the updated data. 
+* Show total count of pools present in a ceph cluster.
+
+=== Alternatives
+
+Generate list component can be created and used for listing the pool. But because of its complexity and time constraint, we will pick this up later.
+
+=== Data model impact:
+
+
+=== Impacted Modules:
+
+==== Tendrl API impact:
+
+/api/1.0/GetPoolList
```

Comment:
This part lack information. state what is available currently(cluster wide list API available), what extra information reqd in the current API. Also specify the any extra APIs required(syatem wide listing?)
Which component implements sorting and filtering? 
Any APis required to get the utlization data?